lib/services/apache.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Package for apache2 service tests
+#
+# Maintainer: Huajian Luo <hluo@suse.com>
+
+package services::apache;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+sub install_apache {
+    zypper_call('in apache2');
+}
+
+sub enable_apache {
```

Comment:
this is a minor thing, but if we call start_apache_service, we could also name this function enable_apache_service. Or the other function just start_apache. 

I would even go further and remove "apache" from all functions since it's implicit in the call. So, we could have something like this:

  ```
  services::apache::install_service();
  services::apache::enable_service();
  services::apache::start_service();
  services::apache::check_service();
  services::apache::check_function();
```